nuget/lib/dependabot/nuget/file_updater.rb

```diff
@@ -62,7 +62,8 @@ def try_update_projects(dependency)
 
           next unless project_dependencies.any? { |dep| dep.name.casecmp(dependency.name).zero? }
 
-          NativeHelpers.run_nuget_updater_tool(repo_contents_path, proj_path, dependency, !dependency.top_level?)
+          NativeHelpers.run_nuget_updater_tool(repo_contents_path, proj_path, dependency, !dependency.top_level?,
+                                               credentials)
```

Comment:
Not necessarily for this PR, just general advice:

Ruby methods with more than 2 or 3 positional parameters get out of hand easily, because it's not easy to rememeber the order of parameters so you need to check signature every time and makes it easy to make mistakes.

I recommend to use keyword parameters, like `credentials: credentials`.

Reply:
Added commit with the change.